web_scraping.py

```
@@ -0,0 +1,92 @@
+        return headers, data
+    except Exception as e:
+        print(f"Error parsing the table: {e}")
+        return None, None
+
+def save_to_csv(headers, data, filename="helicopter_prison_escapes.csv"):
+    try:
+        print("Saving data to CSV...")
+        df = pd.DataFrame(data, columns=headers)
+        df.to_csv(filename, index=False)
+        print(f"Data successfully saved to {filename}")
+        return df
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)